### PR TITLE
21332 Dissolutions job - Update filer config

### PR DESCRIPTION
**Pull request by leodube-aot**
*Issue #:* /bcgov/entity#21332

*Description of changes:*
- Quick fix to update the config for the filer connection.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: jobs/involuntary-dissolutions/Makefile [hunk 1]
```diff
@@ -45,7 +45,7 @@ build-req: clean ## Upgrade requirements
 	cat requirements/bcregistry-libraries.txt >> requirements.txt ;\
 	pip install -Ur requirements/bcregistry-libraries.txt
 
-install: clean ## Install python virtrual environment
+install: clean ## Install python virtual environment
```
Comment (leodube-aot) [inline jobs/involuntary-dissolutions/Makefile:48]: Just noticed this typo is on pretty much every Makefile. I can fix them all at once in this PR if that makes sense.

File: jobs/involuntary-dissolutions/involuntary_dissolutions.py [hunk 2]
```diff
@@ -119,9 +118,8 @@ async def put_filing_on_queue(filing_id: int, app: Flask, qsm: QueueService):
         await qsm.publish_json_to_subject(payload, subject)
     except Exception as err:  # pylint: disable=broad-except # noqa F841;
         # mark any failure for human review
-        capture_message(
-            f'Queue Error: Failed to place filing {filing_id} on Queue with error:{err}',
-            level='error'
+        app.logger.error(
+            f'Queue Error: Failed to place filing {filing_id} on Queue with error:{err}'
```
Comment (leodube-aot) [inline jobs/involuntary-dissolutions/involuntary_dissolutions.py:122]: Not 100% what `capture_message` does, but I wasn't seeing any output to the terminal, causing `put_filing_on_queue` to fail silently. The reason it was failing is that the NATS_FILER_SUBJECT was not set in the config.

Comment (argush3) [reply jobs/involuntary-dissolutions/involuntary_dissolutions.py:122]: The `capture_message` is for sentry.

Lets leave the `capture_message` and add the `app.logger.error` statement as well.

File: jobs/involuntary-dissolutions/Makefile (hunk 1 above)
Comment (argush3) [reply jobs/involuntary-dissolutions/Makefile:48]: yeah, that'd be great.  Let's fix all the typos.  thanks!

File: jobs/involuntary-dissolutions/involuntary_dissolutions.py (hunk 2 above)
Comment (leodube-aot) [reply jobs/involuntary-dissolutions/involuntary_dissolutions.py:127]: Added back the capture_message

File: queue_services/entity-filer/Makefile [hunk 3]
```diff
@@ -45,7 +45,7 @@ build-req: clean ## Upgrade requirements
 	pip install pip==24.0 ;\
 	pip install wheel==0.43.0 ;\
 	pip install -Ur requirements/prod.txt ;\
-	pip freeze | sort > requirements.txt ;\
+	pip freeze | sort > requirementsvirtual
```
Comment (argush3) [inline queue_services/entity-filer/Makefile:48]: accidental i'm thinking?

Comment (leodube-aot) [reply queue_services/entity-filer/Makefile:48]: Good catch, not sure how that happened

**Review by argush3 — APPROVED** [review]

**Pull request merged**